src/hooks/usePagination.ts

```diff
@@ -13,12 +13,15 @@ export const usePagination = (perPage: number) => {
     setPage(() => newPage)
   }
 
-  const getTotalPages = (totalItems: string | null) => {
+  const getTotalPages = (totalItems: string | number | null) => {
     if (!totalItems) {
```

Comment:
I still think it kinda makes sense to explicitly check for null here. Cause having zero pages of stuff vs having no knowledge of how many pages there are because filters are used - are different situations
In CLI repo I have a lint rule that doesn't allow this ! casting and requires explicit checks

Reply:
Fixed